RELEASE_NOTES

```diff
@@ -1,3 +1,17 @@
+========================================================================
+Amazon CloudWatch Agent 1.300062.0 (2025-11-05)
+========================================================================
+Features:
+* [ContainerInsights] Enable opt-in for high frequency GPU metrics collection with configurable intervals for ML inference workloads
+
+Enhancements:
+* [Logs] Fallback to iterative DescribeLogGroups if batch API not supported in region
+* [ContainerInsights] Update OTel contrib dependencies to add enhanced CI checks and nil checks for neuron decorators
```

Comment:
nit: You can leave out the `Update OTel contrib dependencies to` part. Its overall just an enhancement of the agent.

`Add enhanced CI checks and nil checks for neuron decorators to prevent panics`